tests/conftest.py

```diff
@@ -9,11 +9,16 @@
 @pytest.fixture(params=[2, 3])
 def simulated_dataset(request):
     """Make a simulated dataset."""
-    size = (128,) * request.param
-    num_blobs = 5
+    size = (256,) * request.param
+    num_blobs = 6
     img, coords = simulate_image(size=size, num_blobs=num_blobs)
-    mask = label(img > 2.3)
+    mask = label(img > 2.2)
+    # remove small objects
+    tmp = np.unique(mask, return_counts=True)
+    if any(tmp[1] < 5):
+        for val in tmp[0][tmp[1] < 5]:
+            mask[mask == val] = 0
+        mask = label(mask)
```

Comment:
You could consider using this:
https://scikit-image.org/docs/stable/api/skimage.morphology.html#skimage.morphology.remove_small_objects

Reply:
Done